libdnf/rpm/repo_sack.cpp

```diff
@@ -30,4 +35,93 @@ RepoWeakPtr RepoSack::new_repo(const std::string & id, Repo::Type type) {
     return add_item_with_return(std::move(repo));
 }
 
+static void load_config_from_parser(
```

Comment:
The code looks nearly identical to static void load_config_from_parser() in base.cpp. What do you think about unification of both functions?